news/2024/2024-12-12-osu-mania-7k-world-cup-2025-registrations-now-open.md

```diff
@@ -0,0 +1,57 @@
+
+Now, moving on to the bracket format, we are going to monitor the amount of signups before deciding if qualifiers will seed into a traditional Round of 32 bracket, or last year's Round of 24, so make sure to tell all your friends to press the shiny pink button to register [here](https://osu.ppy.sh/community/tournaments/48)!
+
+For a refresher of what a Round of 24 bracket would look like, look no further:
+
+![](/wiki/Tournaments/MWC/2024_7K/img/mwc7k2024-bracket.png)
+
+As you can see, seeds 1 through 8 get a one week break before playing in the Round of 16. Please refer to last year's [Challonge bracket](https://challonge.com/MWC7K_2024) for the full picture.
+
+As always, for more details pertaining to the tournament, refer to the **[osu!mania 7K World Cup 2025 wiki article](/wiki/Tournaments/MWC/2025_7K)**. You can also discuss this event in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/2014134)** or the public **[community server](https://discord.gg/0Vxo9AsejDkGlk3H)**.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | Dec 12 (15:00 UTC) – Dec 26 (23:59 UTC) |
+| Qualifier showcase | Jan 5 (14:00 UTC) |
+| Qualifier stage | Jan 11 & Jan 12 |
+| Round of 24 | Jan 18 & Jan 19 |
+| Round of 16 | Jan 25 & Jan 26 |
+| Quarterfinals | Feb 1 & Feb 2 |
+| Semifinals | Feb 8 & Feb 9 |
+| Finals | Feb 15 & Feb 16 |
+| Grand Finals | Feb 22 & Feb 23 |
+
+## Prizes
+
+The osu!mania 7K World Cup 2025 offers a $1,200 prize pool, along with limited-edition merch!
+
+- **1st:** 50% of the prize pool, single-run merch, profile badge, **osu!mania 7K Champion** user title for one year
+- **2nd:** 30% of the prize pool, single-run merch, profile badge
+- **3rd:** 20% of the prize pool, single-run merch, profile badge
+
+## How do I register?
+
+**[Click here!](https://osu.ppy.sh/community/tournaments/48)**
+
+Please note that teams will be formed by each country's captain *after* the registration phase has concluded. **The deadline for registering is December 26, 23:59 UTC. Make sure to submit yours before then!**
+
+*Notice: you may only participate if your position in the osu!mania 7K global ranking is #5,000 or higher, and you have not violated the [osu! community rules](/wiki/Rules) in the past 12 months.*
+
+If your registration passes our internal checks, you will be placed on the candidate list for your country. When that country's captain has been assigned, the list will be sent to them. This may take a while, so please be patient.
+
+*Remember that any registrations submitted after the deadline will not be considered under any circumstances — submit your registration ahead of time to avoid any last-minute headaches. If you run into any issues, send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).*
```

Comment:
```suggestion
*Remember that any registrations submitted after the deadline will not be considered under any circumstances — submit your registration ahead of time to avoid any last-minute headaches. If you run into any issues, email [tournaments@ppy.sh](mailto:tournaments@ppy.sh).*
```